_posts/2018-04-06-all-hands.md

```diff
@@ -0,0 +1,182 @@
+ ---
+layout: post
+title: "The Rust Team All Hands in Berlin: a Recap"
+author: Aaron Turon
+---
+
+Last week we held an "All Hands" event in Berlin, which drew more than 50 people
+involved in 15 different Rust Teams or Working Groups, with a majority being
+volunteer contributors. This was the first such event, and its location reflects
+the current concentration of team members in Europe. The week was a smashing
+success which we plan to repeat on at least an annual basis.
+
+The impetus for this get-together was, in part, our [ambitious plans] to ship
+*Rust, 2018 edition* later this year. A week of facetime was great a great way
+to kick off these efforts!
+
+We've also asked attendees to blog and tweet about their experiences at the
```

Comment:
I feel like it should be mentioned that this was a work-driven meeting, which is why the rest of the blog post turns out to be a list of work items.